crates/flox-rust-sdk/src/models/publish.rs

```diff
@@ -1196,4 +1224,14 @@ mod tests {
 
         assert_eq!(snapshot_actual, snapshot_actual);
     }
+
+    #[tokio::test]
+    async fn resolves_indirect_ref_to_gitfile() {
```

Comment:
Do you mean githttps?

Reply:
Yeah, it was originally just a test to make sure I didn't get a `path` reference and I missed updating the name.